analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SillyBitwiseOperation.Fixed.cs

```diff
@@ -13,7 +13,7 @@ public enum Types
 
     class SillyBitwiseOperation
```

Comment:
do we already have a test like below?
```
    public void Repro(object[] args)
    {
        var fail = false;
        bool x;
        foreach (var arg in args)
        {
            x = fail | !CheckArg(arg);   // Noncompliant?
        }
    }
```

Reply:
Yes, we do. This behavior is equivalent to cases with `unchanged` variable.